### DISCUSSION TITLE
Document the need to add `custom.regex` to `enabledManagers` in the custom manager documentation article

### DISCUSSION THREAD

**Discussion by blakeNaccarato** [renovatebot/renovate#30124] [Suggest an Idea]
### Tell us more.

While this advice is given in https://docs.renovatebot.com/configuration-options/#enabledmanagers, it is not given in https://docs.renovatebot.com/modules/manager/regex/, which just caused a few hours of headache 😅 constantly tweaking a regex only to discover it's not enabled.

Even though the user's mental model of Renovate should always entail "Enable it in `enabledManagers` before using it", the possibly unintuitive `custom.regex` name and the magical-ness of `customManagers` confused me here.

**Comment by rarkins** [comment]
You don't need to define enabledManagers at all. If you do, make sure you include the ones you're using.

Feel feee to propose a PR with your suggestion

**Comment by blakeNaccarato** [comment]
Ah! Okay I must've started using `enabledManagers` awhile back and come to the misunderstanding that it's a required part of setup. I see that detailed now in the "Managers" page, thanks for pointing it out.

I'll be sure to keep that in mind and use appropriate language if I draft a clarification in the custom managers docs.
